app/models/search_tab.rb

```diff
@@ -8,11 +8,12 @@
 # the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
+# SearchTab
 class SearchTab
   TABS = %w(All Images Audio Videos Sets)
   attr_reader :tab
 
-  def initialize(tab='All')
+  def initialize(tab = 'All')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.